find_primes.py

```diff
@@ -1 +1,29 @@
+#num1 is the original number, num2 is the prime number, num3 is all of the primes,
+num1 = int(input("Enter any positive integer: "))
+num2 = num1 + 1
+num3 = []
+count = 0
+
+def find_prime(a):
+  '''find_prime(a) -> yes or no
+  Prints whether a is a prime'''
```

Comment:
This function actually returns whether or not the input is prime, rather than printing it.